### ISSUE TITLE
Need to add disclaimer about using SSL when using swagger-ui for the public REST API

### ISSUE THREAD

**Issue by mattcarabine** [couchbaselabs/couchbase-mobile-portal#522]
Just like the CORS info, we need to add info stating that in order to use the swagger-ui correctly SSL must be configured.

Additionally we need to see if there is a way for swagger-ui to force using https (as opposed to http as it does atm).

The admin+cblite REST API pages can be left as is as you cannot directly access them anyway.

**Comment by Prudwell** [comment]
In regards to forcing https, I might have some insight. I just spent some hours tracking down an issue with getting https working with the public spec and discovered that the spec doesn't even allow https in its schemes.

I put in a [ticket](591) for that but in my research I learned that swagger client (used by swagger UI) can manually have scheme set in the constructor and that may work in swagger UI.

```
         publicClient = new Swagger({
            url: constants.API_ENDPOINT_PROTOCOL_ENV + constants.WEB_DOMAIN_WITH_SUBDOMAIN + '/public.json',
            usePromise: true,
            scheme: "https"
        })
```

Swagger client will also [try a few ways](https://github.com/swagger-api/swagger-js/blob/master/lib/client.js#L345) to detect on its own what scheme to use if it isn't manually set. I believe it looks at the url of the spec file and if that is loaded via https. If that was not loaded with a scheme, it will look at the window url if it's being run in a browser to try to get the current scheme that way.
